### PR TITLE
feat: remove guides, add doc links

### PR DESCRIPTION
**Pull request by samyak-max**
image

Removed guides link from nav and logic for the page (deleted Guides component, guide page, and /guide/[id]), added docs links.

### CODE REVIEW

File: components/Footer.jsx [hunk 1]
```diff
@@ -23,6 +23,36 @@ function Footer() {
...
+          a: true,
+        },
+        {
+          name: 'Developers',
+          link: 'https://elixir-cloud-aai.github.io/guides/guide-dev/',
+          a: true,
+        },
+        {
+          name: 'Administrators',
+          link: ' https://elixir-cloud-aai.github.io/guides/guide-admin/',
```
Comment (git-anurag-hub) [inline components/Footer.jsx:46]: Remove trailing spaces!

**Review by git-anurag-hub — CHANGES_REQUESTED** [review]
Just one thing, give a little bit more padding at the bottom of the footer. Other than that LGTM!

**Review by git-anurag-hub — APPROVED** [review]

**Review by git-anurag-hub — DISMISSED** [review]

**Comment by git-anurag-hub** [comment]
Hey @uniqueg LGTM! 
You can have a look if you want!

**Review by uniqueg — CHANGES_REQUESTED** [review]
Looks great - just one minor detail: Perhaps rather than having a separate "Home" item in the "Docs" section in the footer, you could link the "Docs" section title itself. I know it's not consistent with the behavior of the other section titles, but I think it still makes sense and saves us some space.

**Comment by uniqueg** [comment]
Also, another minor detail: I changed the PR title. Please always use present tense for semantic commit messages.

**Review by uniqueg — APPROVED** [review]

**Pull request merged**